pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "pixi-cuda-environment"
+version = "0.1.0"
+description = "Add a short description here"
+authors = [{name = "Eric Ma", email = "ericmajinglong@gmail.com"}]
+requires-python = ">= 3.11"
+dependencies = ["pandas>=2.2.2,<2.3", "numpy>=2.0.1,<2.1"]
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.pixi.project]
+channels = ["conda-forge"]
+platforms = ["osx-arm64", "linux-64"]
+
+[tool.pixi.pypi-dependencies]
+pixi-cuda-environment = { path = ".", editable = true }
+
+[tool.pixi.tasks]
+test = "pytest"
+
+[tool.pixi.dependencies]
+ruff = ">=0.5.5,<0.6"
+pytest = ">=8.3.2,<8.4"
+ipython = ">=8.26.0,<9"
+jax = "*"
+pytest-cov = ">=5.0.0,<6"
+
+# Feature Definitions
+[tool.pixi.feature.cuda-feature]
```

Comment:
I would remove the "feature" in the name, and maybe call the feature `cuda12` so that another one can be added for a new major cuda release in the future?

Reply:
I like the idea of removing the `-feature` suffix, I originally put it there for pedagogical reasons so that folks don't get confused between "which cuda are we talking about"? But in retrospect, it's more confusing to have the `-feature` and to break 1:1:1:1... mapping of names, a practice that I often advocate for.